### PR TITLE
Aruna/sqli ama alerts

### PR DESCRIPTION
**Pull request by vutran01**
## PR Checklist

* [ ] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.

### If adding or updating templates:
* [ ] post a screenshot of templates and/or gallery changes
* [ ] ensure all steps have meaningful names
* [ ] ensure all parameters and grid columns have display names set so they can be localized
* [ ] ensure that parameters id values are unique __or they will fail PR validation__ (parameter ids are used for localization)
* [ ] ensure that steps names are unique __or they will fail PR validation__ (step names are used for localization)
* [ ] grep `/subscription/` and ensure that your parameters don't have any hardcoded resourceIds __or they will fail PR validation__
* [ ] remove `fallbackResourceIds` and `fromTemplateId` fields from your template workbook __or they will fail PR validation__

### CODE REVIEW

File: Workbooks/Workloads-AMA/Alerts/Alerts.workbook [hunk 1]
```diff
@@ -0,0 +1,921 @@
...
+              "value": "2"
+            },
+            "showPin": true,
+            "name": "Alert Summary"
+          },
+          {
+            "type": 3,
+            "content": {
+              "version": "KqlItem/1.0",
+              "query": "{\"version\":\"1.0.0\",\"content\":\"[\\r\\n\\t{\\r\\n\\t\\t\\\"severity\\\": \\\"Sev0\\\"\\r\\n\\t},\\r\\n\\t{\\r\\n\\t\\t\\\"severity\\\": \\\"Sev1\\\",\\r\\n\\t\\t\\\"new\\\": 0,\\r\\n\\t\\t\\\"acknowledged\\\": 0,\\r\\n\\t\\t\\\"closed\\\": 0\\r\\n\\t},\\r\\n\\t{\\r\\n\\t\\t\\\"severity\\\": \\\"Sev2\\\"\\r\\n\\t},\\r\\n\\t{\\r\\n\\t\\t\\\"severity\\\": \\\"Sev3\\\"\\r\\n\\t},\\r\\n\\t{\\r\\n\\t\\t\\\"severity\\\": \\\"Sev4\\\"\\r\\n\\t}\\r\\n]\",\"transformers\":[{\"type\":\"jsonpath\",\"settings\":{\"columns\":[{\"path\":\"$.severity\",\"columnid\":\"Severity\"}]}}]}",
```
Comment (yves-chan) [inline Workbooks/Workloads-AMA/Alerts/Alerts.workbook:333]: nit: don't need anything other than severity column

**Review by yves-chan — APPROVED** [review]
Looks good, just had one nit

File: Workbooks/Workloads-AMA/Alerts/Alerts.workbook (hunk 1 above)
Comment (vutran01) [reply Workbooks/Workloads-AMA/Alerts/Alerts.workbook:333]: yeah, i'm just using the severity column to merge/show empty severity. the others could be removed, however, i'm still trying to get the links to show up on the no data ones

**Review by yves-chan — APPROVED** [review]
LGTM

**Pull request merged**